sssom/cli.py

```diff
@@ -139,9 +139,22 @@ def convert(input: str, output: str, output_format: str):
     help="Defines wether the curie map in the metadata should be extended or replaced with "
     "the SSSOM default curie map. Must be one of metadata_only, sssom_default_only, merged",
 )
+@click.option(
+    "-p",
+    "--clean-prefixes",
+    default=True,
+    is_flag=True,
```

Comment:
Where is this parameter (`is_flag`) going? Shouldn't this be part of `parse` below?

Reply:
`is_flag` means that you can just say `--clean-prefixes` instead of `--clean-prefixes true`. Does this make sense?